### PR TITLE
UA review of usage.md

### PR DESCRIPTION
**Pull request by jhornik**
UA review.

Before submitting a pull request, Please make sure the following:

## Description

Please explain the changes you made.

Link to design (if applicable)

### Checklist
- [ ] Code compiles correctly
- [ ] Relevant tests were added (unit / contract / integration)
- [ ] Relevant logs were added
- [ ] Formating and linting run locally successfully
- [ ] All tests passing
- [ X] UA review
- [ ] Design is documented
- [ ] Extended the README / documentation, if necessary
- [ ] Open source is approved

### CODE REVIEW

File: docs/docs/usage.md [hunk 1]
```diff
@@ -7,58 +7,59 @@
...
 ### How to build an MTA archive from the project sources
 
 #### Prerequisites
-* `GNU Make 4.2.1` installed in your build environmnet. 
-* module build tools
+* `GNU Make 4.2.1` is installed in your build environment. 
+* Module build tools are installed in your build environment.
 
-See the correponding `Download and Installation` sections for more details.
+For more information, see the corresponding `Download` and `Installation` sections.
```
Comment (jhornik) [inline docs/docs/usage.md]: I recommend inserting links to the  mentioned Download and Installation sections.

File: docs/docs/usage.md [hunk 2]
```diff
@@ -7,58 +7,59 @@
 | help    | `mbt [command] --help` or<br> `mbt [command] -h`    | Prints detailed information on the selected command.                           
 <br>
 
-### How to know the tool's version
+### How to display the current tool version
```
Comment (katunin-ng) [inline docs/docs/usage.md]: I do not quite like the "display" here.  Users do not want to display the version. They want to know what the version of the installed tool is.

Comment (jhornik) [reply docs/docs/usage.md]: I understand. But "how to know a version" would be better rephrased. You are actually displaying it (printing) when you run the command. Maybe better: "How to find out the tool version"

Comment (katunin-ng) [reply docs/docs/usage.md]: Like it. I will update.

Comment (jhornik) [reply docs/docs/usage.md]: Ok, thanks.

**Pull request merged**